src/CheckBoxFilter.xml

```diff
@@ -0,0 +1,82 @@
+            <description>The attribute to apply filter when filtering by 'attribute'. Note: for enum use the key value, for boolean use 'true' or 'false'</description>
+            <attributeTypes>
+                <attributeType name="String"/>
+                <attributeType name="Integer"/>
+                <attributeType name="Decimal"/>
+                <attributeType name="Boolean"/>
+                <attributeType name="Enum"/>
+            </attributeTypes>
+        </property>
+        <property key="attributeValue" type="string" required="false">
+            <caption>Attribute value</caption>
+            <category>Checked</category>
+            <description>Value of the attribute to be matching (used when filtering on attribute)</description>
+        </property>
+        <property key="constraint" type="entityConstraint" required="false" entityProperty="listviewEntity">
+            <caption>XPath constraint</caption>
+            <category>Checked</category>
+            <description>Constraint on the data (used when filtering by XPath)</description>
+        </property>
+        <property key="unCheckedFilterBy" type="enumeration" defaultValue="none">
+            <caption>Filter by </caption>
+            <category>Unchecked</category>
+            <description>Select the method of filtering</description>
+            <enumerationValues>
+                <enumerationValue key="none">None</enumerationValue>
+                <enumerationValue key="attribute">Attribute</enumerationValue>
+                <enumerationValue key="XPath">XPath</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="unCheckedAttribute" type="attribute" required="false" entityProperty="listviewEntity">
+            <caption>Attribute </caption>
+            <category>Unchecked</category>
+            <description>The attribute to apply filter when filtering by 'attribute'. Note: for enum use the key value, for boolean use 'true' or 'false'</description>
+            <attributeTypes>
+                <attributeType name="String"/>
+                <attributeType name="Integer"/>
+                <attributeType name="Decimal"/>
+                <attributeType name="Boolean"/>
+                <attributeType name="Enum"/>
+            </attributeTypes>
+        </property>
+        <property key="unCheckedAttributeValue" type="string" required="false">
+            <caption>Attribute value </caption>
+            <category>Unchecked</category>
+            <description>Value of the attribute to be matching (used when filtering on attribute)</description>
+        </property>
+        <property key="unCheckedConstraint" type="entityConstraint" required="false" entityProperty="listviewEntity">
+            <caption>XPath constraint </caption>
+            <category>Unchecked</category>
+            <description>Constraint on the data (used when comparison is XPath)</description>
```

Comment:
Description should be equal as 'checked' one
``` xml
<description>Constraint on the data (used when filtering by XPath)</description>
```

Reply:
Thanks